### PR TITLE
Add Courses and Interactive tutorials (RU)

### PR DESCRIPTION
**Pull request by MaXXX74**
## What does this PR do?
Added several interactive resources for learning Python and JavaScript Russian users.

### How do we know it's really free?
Because it's true.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ru.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Cодержание
+﻿### Cодержание
```
Comment (davorpa) [inline courses/free-courses-ru.md:1]: Where is the change here?

File: more/free-programming-interactive-tutorials-ru.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
-### Index
+﻿### Index
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-ru.md:1]: Where is the change here?

File: more/free-programming-interactive-tutorials-ru.md [hunk 3]
```diff
@@ -18,8 +19,14 @@
 * [Обучение git при помощи визуализации](https://learngitbranching.js.org/?locale=ru_RU)
 
 
+### JavaScript
+
+* [CodeCombat - игры с программированием для изучения Python и JavaScript](http://codecombat.com)
```
Comment (davorpa) [inline more/free-programming-interactive-tutorials-ru.md:24]: Codecombat is already as global resource in "ProblemSets & Competitive Programming" 

https://github.com/EbookFoundation/free-programming-books/search?q=codecombat

HTTPS is always better than HTTP

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-ru.md (hunk 1 above)
Comment (MaXXX74) [reply courses/free-courses-ru.md]: fixed

File: more/free-programming-interactive-tutorials-ru.md (hunk 2 above)
Comment (MaXXX74) [reply more/free-programming-interactive-tutorials-ru.md]: fixed

File: courses/free-courses-ru.md [hunk 4]
```diff
@@ -148,6 +148,9 @@ ADV - Продвинутый. Тонкости.
 ### Python
 
 * [Автоматизация тестирования с помощью Selenium и Python](https://stepik.org/course/575) - Stepik (INT)
+* [Добрый, добрый Python - обучающий курс от Сергея Балакирева](https://stepik.org/course/100707) - Stepik (BEG)
```
[the remark this answers is not in the document]
Comment (MaXXX74) [reply courses/free-courses-ru.md]: fixed

File: more/free-programming-interactive-tutorials-ru.md (hunk 2 above)
Comment (MaXXX74) [reply more/free-programming-interactive-tutorials-ru.md]: fixed

File: more/free-programming-interactive-tutorials-ru.md (hunk 3 above)
Comment (MaXXX74) [reply more/free-programming-interactive-tutorials-ru.md]: fixed

**Review by MaXXX74 — COMMENTED** [review]
I've fixed them all

**Review by davorpa — APPROVED** [review]
Thanks for completing the remaining authoring in Python "stepicks"

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**